### ISSUE TITLE
ImageBackground : remind developer for different RN version

### ISSUE THREAD

**Issue by samuelplaymore** [kfiroo/react-native-cached-image#84]
we use RN 0.45, and we got error of undefined component about ImageBackground.

Please handle it, we suggest choose one way in the following.

1. update README and tell developer use different version, such as 1.3.5 for RN below 0.45
2. check if ImageBackground is undefined in CacheImage.js, and use Image instead

Thanks :)

**Comment by kfiroo** [comment]
@samuelplaymore Hey!
You are absolutly right, I will update the README to reflect that.

The recomended solution to your issue is to use the [`renderImage`](https://github.com/kfiroo/react-native-cached-image#props) prop of `CachedImage` to render your own `ImageComponent`.
That may be a RN `Image` or any other image component such as https://github.com/expo/react-native-responsive-image

**Comment by kytwb** [comment]
@kfiroo why not start using semantic versioning and make a major up for when there's breaking changes like this?

**Comment by kfiroo** [comment]
@kytwb True, I thought the minor version bump was enough but a second look proved me wrong :)
Should we publish a new `2.0.0` version for this or wait for the next big change?

**Comment by exos** [comment]
To fix:

```
Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in. Check the render method of `CachedImage`.
```
Next return other warning:

```
Possible Unhandled Promise Rejection (id: 0):
Cannot read property 'getHostNode' of null
TypeError: Cannot read property 'getHostNode' of null
```

You can do:

```jsx
                <CachedImage
                    source={{
                        uri: this.props.uri,
                    }}
                    renderImage={(data) => {
                        return (<Image {...data}/>);
                    }}
                />
```

https://github.com/kfiroo/react-native-cached-image/blob/35b361695de37e4f72668c6c9c227e62ee3a87bb/CachedImage.js#L55
